src/utilities.c

```diff
@@ -0,0 +1,49 @@
+A #include<stdio.h>
+#include <ctype.h>
+#include <string.h>
+
+    /* helper function: read a line and then return
+       the first string with whitespace stripped off */
+    static int get_a_line(FILE *fp, char *buf) {
```

Comment:
check indentation. Run: `clang-format -i --style=google`

Remove the static keyword